### PR TITLE
Add Finnish translation

### PR DESCRIPTION
**Pull request by ArikSquad**
**Finnish** translation to **Sol Client**

### CODE REVIEW

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+sol_client.mod.chat.option.colours=Värit
+sol_client.mod.chat.option.width=Leveys
+sol_client.mod.chat.option.closedHeight=Suljettu korkeus
+sol_client.mod.chat.option.openHeight=Avoin korkeus
+sol_client.mod.chat.option.links=Linkit
+sol_client.mod.chat.option.promptLinks=Pikalinkit
+sol_client.mod.chat.option.chatFilter=Ota chat-suodatin käyttöön
+sol_client.mod.chat.option.filteredWordsContent=Suodatetut Sanat
+
+sol_client.mod.tab_list.name=Tab List
```
Comment (TheKodeToad) [inline game/src/main/resources/assets/sol_client/lang/fi_FI.lang:172]: Still in English. Not sure if you'd want to translate this though.

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+
+sol_client.mod.crosshair.option.thirdPerson=Näytä Kolmannessa Persoonassa
+sol_client.mod.crosshair.option.spectatorAlways=Näytä katsojatilassa
+sol_client.mod.crosshair.option.debug=Näytä virheenkorjauspeittokuvassa
+sol_client.mod.crosshair.option.blending=Sekoitus
+sol_client.mod.crosshair.option.crosshairColour=Väri
+sol_client.mod.crosshair.option.highlightEntities=Korosta Entiteetit
+sol_client.mod.crosshair.option.entityColour=Entiteettien Väri
+
+sol_client.mod.scoreboard.name=Scoreboard
```
Comment (TheKodeToad) [inline game/src/main/resources/assets/sol_client/lang/fi_FI.lang:213]: Again, English.

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+sol_client.mod.zoom.option.key=Näppäin
+sol_client.mod.zoom.zoom_out=Zoomaa ulos
+sol_client.mod.zoom.option.zoomOutKey=Ulospäin Zoomaamisen näppäin
+sol_client.mod.zoom.zoom_in=Zoomaa sisään
+sol_client.mod.zoom.option.zoomInKey=Sisäänpäin Zoomaamisen näppäin
+sol_client.mod.zoom.option.cinematic=Elokuvakamera
+sol_client.mod.zoom.option.reduceSensitivity=Vähennä herkkyyttä
+sol_client.mod.zoom.option.scrolling=Vieritä Zoomataksesi
+sol_client.mod.zoom.option.smooth=Zoomaus Animaatio
+sol_client.mod.zoom.option.factor=Zoomaustekijä
```
Comment (TheKodeToad) [inline game/src/main/resources/assets/sol_client/lang/fi_FI.lang:279]: Name in English, but keys not.

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+
+sol_client.mod.replay.name=Replay
```
Comment (TheKodeToad) [inline game/src/main/resources/assets/sol_client/lang/fi_FI.lang:281]: English.

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang [hunk 5]
```diff
@@ -0,0 +1,442 @@
...
+sol_client.mod.replay.interpolator.cubic=Cubic Spline
+sol_client.mod.replay.interpolator.linear=Lineaarinen
+
+sol_client.mod.replay.option.showServerIPs=Näytä palvelimien IP-osoitteet
+
+sol_client.mod.quickplay.name=Quick Play
+sol_client.mod.quickplay.description=Liity nopeasti pelien jonoihin näppäimistön avulla.
+
+sol_client.mod.quickplay.key=Quick Play
+sol_client.mod.quickplay.option.menuKey=Näppäin
```
Comment (TheKodeToad) [inline game/src/main/resources/assets/sol_client/lang/fi_FI.lang:313]: Names in English.

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang [hunk 6]
```diff
@@ -0,0 +1,442 @@
...
+sol_client.mod.hypixel_util.option.lobbySoundsVolume=Aulan äänenvoimakkuus
+sol_client.mod.hypixel_util.option.housingMusicVolume=Housing Musiikin Äänenvoimakkuus
+sol_client.mod.hypixel_util.option.popupEvents=Popup Tapahtumat
+sol_client.mod.hypixel_util.option.autogg=Auto GG
+sol_client.mod.hypixel_util.option.autoggMessage=GG viesti
+sol_client.mod.hypixel_util.option.hidegg=Piilota GG
+sol_client.mod.hypixel_util.option.autogl=Auto GL
+sol_client.mod.hypixel_util.option.autoglMessage=GL Viesti
+sol_client.mod.hypixel_util.option.hidegl=Piilota GL
+sol_client.mod.hypixel_util.option.levelhead=Level Head
```
Comment (TheKodeToad) [inline game/src/main/resources/assets/sol_client/lang/fi_FI.lang:328]: Maybe you could translate level head and auto.

**Review by TheKodeToad — CHANGES_REQUESTED** [review]
Looks (almost) good to me!

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang (hunk 1 above)
Comment (ArikSquad) [reply game/src/main/resources/assets/sol_client/lang/fi_FI.lang:172]: I don't want to, 100% this is more understandable

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang (hunk 2 above)
Comment (ArikSquad) [reply game/src/main/resources/assets/sol_client/lang/fi_FI.lang:213]: again this is only right way

File: game/src/main/resources/assets/sol_client/lang/fi_FI.lang (hunk 6 above)
Comment (ArikSquad) [reply game/src/main/resources/assets/sol_client/lang/fi_FI.lang:328]: I think translating Level Head would lose the meaning. Auto GG and Auto GL too.

**Comment by TheKodeToad** [comment]
I would re-review but I want the YOLO badge even if it's unlikely that this "counts".

**Pull request merged**